manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OrbxSimulationSystems.OrbxCentral
+PackageVersion: 4.1.42.821
+PackageLocale: en-US
+Publisher: Orbx Simulation Systems Pty Ltd
+PackageName: Orbx Central
+License: Copyright © 2021 Orbx Simulation Systems Pty Ltd

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate